e2e/tests/mobile/registered-shopper.spec.js

```diff
@@ -153,7 +153,7 @@ test("Registered shopper can add item to wishlist", async ({ page }) => {
   if(!isLoggedIn) {
     await registerShopper({
       page,
-      userCredentials: REGISTERED_USER_CREDENTIALS,
+      userCredentials: generateUserCredentials(),
```

Comment:
While each individual test runs in isolation from the other tests, we must be mindful that the tests are running against a real deployed instance and the global variables in the test file like `REGISTERED_USER_CREDENTIALS ` will not reset for each test.

When `REGISTERED_USER_CREDENTIALS` is reused for sign up in multiple tests, all tests after the first one will fail since the user with `REGISTERED_USER_CREDENTIALS` will already exist on the instance. Hence if you have signup flows in multiple tests, you must always call `generateUserCredentials()` to avoid conflicting usernames.